assets/js/custom/ProgramSection.js

```diff
@@ -0,0 +1,28 @@
+export class ProgramSection {
```

Comment:
I think we should rename this to be more specific to showing custom translation, or even merge this into CustomTranslationApi, once we don't have the cookie parsing chunk

Reply:
I ended up removing the class since it was more code than just calling the API class directly. We can add something like this when we want to merge some of the repeated code in the sections